src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.cs

```diff
@@ -1032,7 +1032,7 @@ public bool TakeStartLine(ReadableBuffer buffer, out ReadCursor consumed, out Re
             var result = _parser.ParseRequestLine(this, buffer, out consumed, out examined);
             if (!result && overLength)
             {
-                RejectRequest(RequestRejectionReason.RequestLineTooLong);
+                RequestRejectionUtilities.RejectRequest(RequestRejectionReason.RequestLineTooLong);
```

Comment:
IMO the code is more readable if it reads like `throw RequestRejectionUtilities.RejectRequest(reason)`. How would this affect the jit?